swep1rcr.exe/hotkey.c

```diff
@@ -0,0 +1,456 @@
+      }
+    }
+
+  } else if ( (sub_485880(DIK_RMENU, 0) || sub_485880(DIK_LMENU, 0)) && sub_485880(DIK_H, 0) ) {
+
+    // Alt (Right or Left) + H
+
+    if ( dword_4D79EC )
+      goto LABEL_146;
+    dword_4D79EC = 1;
+
+    //FIXME: What does this do?!
+    dword_EC83C8 = 1;
+  } else if ( dword_50C07C == 2 ) {
+
+    // We are in the pause menu??? (or in a race???)
+    //FIXME: Not sure
+
+    if ( sub_485880(DIK_LCONTROL, 0) || sub_485880(DIK_RCONTROL, 0) ) {
+
+      // Control (Left or Right)
+
+      if ( sub_485880(DIK_F10, 0) ) {
+
+        // F10
+
+        if ( dword_4D79EC )
+          goto LABEL_146;
+
+        //FIXME: What does this do?
+        if ( !dword_4D5E00 ) {
+          dword_50C048 ^= 4u;
+          sub_440550(40);
+        }
+
+        dword_4D79EC = 1;
+      }
+
+    } else if ( sub_485880(DIK_MULTIPLY, 0) && sub_485880(DIK_T, 0) ) {
+
+      // Multiply (Numpad) + T
+
+      //FIXME: What does this do?
+      if ( !dword_4D5E00 ) {
+        if ( dword_4D79EC )
+          goto LABEL_146;
+
+        // Make the game very fast?!
+        dword_50CB68 ^= 1u;
+        sub_440550(44);
+
+        dword_4D79EC = 1;
+      }
+    } else if ( sub_485880(DIK_I, 0) && sub_485880(DIK_F, 0) && sub_485880(DIK_L, 0) && sub_485880(DIK_Y, 0)) {
+
+      // I + F + L + Y
+
+        //FIXME: What does this do?
+      if (!dword_4D5E00 ) {
+        if ( dword_4D79EC )
+          goto LABEL_146;
+
+        //FIXME: What does this do?
+        dword_50C048 ^= 0x2000u;
+        sub_440550(43);
+
+        dword_4D79EC = 1;
+      }
+    }
+
+  } else if ( sub_485880(DIK_LCONTROL, 0) || sub_485880(DIK_RCONTROL, 0) ) {
+
+    // Control (Left or Right)
+
+    if ( sub_485880(DIK_J, 0) ) {
+
+      // J
+
+      if ( dword_4D79EC )
+        goto LABEL_146;
+      dword_4D79EC = 1;
+
+      if ( !dword_4B294C ) {
+
+        v2 = sub_421360(aMondotextH0035);
+        strcat(&byte_4D78D0, v2);
+
+      } else {
+
+        if ( dword_4B2944 ) {
+          strcat(&byte_4D78D0, sub_421360(aMondotextH0364));
+          dword_4D79D0 = dword_4B2910;
+          if ( dword_4B2910 )
+            sub_40A120(1);
+          dword_4B2910 = 0;
+        } else {
+          strcat(&byte_4D78D0, sub_421360(aMondotextH0363));
+          dword_4B2910 = dword_4D79D0;
+        }
+        sub_440550(70);
+        dword_4B2944 = dword_4B2944 == 0;
+        if ( !dword_4B2940 ) {
+          v3 = sub_414D90(0, 24);
+          sub_40B740(0, v3);
+        }
+
+        v4 = sub_411810();
+        if ( dword_4B2940 || !v4 || (*(_DWORD *)(v4 + 28) != 24)) {
+          sub_406080((int)aCurrent);
+        }
+
+      }
+    } else if ( sub_485880(DIK_LMENU, 0) && *(_DWORD *)(dword_50C454 + 8) == 13 ) {
+
+      // Alt (Left)
+      //FIXME: What's the other condition?
+
+      if ( sub_485880(DIK_RMENU, 0) ) {
+
+        // Alt (Right)
+
+        if ( dword_4D79EC )
+          goto LABEL_146;
+        dword_4D79EC = 1;
+        if ( !dword_4D5E00 )
```

Comment:
Check if we are on the track selection?! That's the only place where the mirror feature works I think. Other instance also.